gateway/src/acme.rs

```diff
@@ -361,3 +363,12 @@ pub enum AcmeClientError {
 }
 
 impl std::error::Error for AcmeClientError {}
+
+impl From<AcmeClientError> for ApiError {
+    fn from(value: AcmeClientError) -> Self {
+        Self {
+            message: value.to_string(),
+            status_code: StatusCode::INTERNAL_SERVER_ERROR.as_u16(),
+        }
+    }
+}
```

Comment:
Is the error message from the acme client relevant to the user here?

Reply:
Yes, this is only trigger by the create acme request, create custom domain and renew custom domain. We trigger all of these and would want as much info as possible. :smile: